components/Shared/AccountCard/Error.jsx

```diff
@@ -24,7 +24,6 @@ const AccountError = ({ errorMsg, onTryAgain }) => (
       <Text>Error</Text>
     </Box>
     <Box>
-      <Title>Ledger Device Problem</Title>
```

Comment:
@as-dr notice this change - idk if this fucks things up for you but not all errors are going to be Ledger related, so I just took this line out. If we want better ledger device errors (now we should recheck) we can just enhance the messages that are passed to this component. That should be done here https://github.com/openworklabs/filecoin-web-wallet/blob/primary/utils/ledger/reportLedgerConfigError.js#L1 so it should be easy to change (maybe we just add Ledger before device in all those messages)

Reply:
This sounds reasonable.

Reply:
@Schwartz10 

A user unplugs their Ledger
Clicks "Create Account"
Gets an error.

If the Ledger stays unplugged &&

If they click on the Error Card's "Try Again" button they are presented with the option to create an account again.

If they click on the card - to try and create another account - but nothing happens.

Shouldn't we display the ErrorCard again?

Reply:
I can't replicate this @as-dr